resources/app.html

```diff
@@ -2,30 +2,10 @@
 <html>
 <head>
     <meta charset='UTF-8'/>
-    <style>
-    body { margin: 0; }
-    #app-loader {
-        position: absolute;
-        top: 50%; left: 50%; width: 100px; height: 100px;
-        margin: -50px 0 0 -50px;
-        pointer-events: none;
-        border-radius: 50%;
-        border: 8px solid rgba(0,128,192, 0.2);
-        border-left: 8px solid #0080c0;
-        animation: rotator 2s infinite linear;
-    }
-    #webapp { transition: opacity .4s; opacity: 0; }
-    #webapp.loaded { opacity: 1; }
-    @keyframes rotator {
-        0%   { transform: rotate(0deg); }
-        100% { transform: rotate(360deg); }
-    }
```

Comment:
Why did you remove the spinner while the app is loaded? Did you think it is just not worth it because the apps are loaded fast enough?

I think the removal of the spinner could have been an PR on its own. Or at least in an own commit, since it also seems easy to separate from the rest.